packages/forms-web-app/src/controllers/appeals/continue.js

```diff
@@ -0,0 +1,28 @@
+const { apiClient } = require('#lib/appeals-api-client');
+
+const appealSubmissionContinueUrls = {
+	1001: '/appeal-householder-decision/task-list',
+	1005: '/full-appeal/submit-appeal/task-list'
+};
+
+exports.get = async (req, res) => {
+	const {
+		query: { appealId, appealType },
+		session: { email }
+	} = req;
+
+	const user = await apiClient.getUserByEmailV2(email);
+	const appeals = await apiClient.getUserAppealsById(user.id);
+
+	const appeal = appeals.find((appeal) => appeal._id === appealId)?.appeal;
+
+	if (!appeal) throw new Error(`Appeal ${appealId} does not belong to user ${user.id}`);
```

Comment:
Should we add an endpoint to get the appeal by ID, instead of reading them all?

Reply:
I did wonder about that but didn't want to meddle too much. Would be a nicer solution if that works for you though

Reply:
done ✅